changelog/3825.bugfix.rst

```diff
@@ -0,0 +1 @@
+This patch fixes query_args in JSOCClient return object. For formating dataset query, .pop method was used for extracting primekeys from input arguments list and that led to invalidating shalow copy of iargs object. This patch forces deepcopy of object so that primekeys persists.
```

Comment:
The changelog should be a user facing description of the change, so in this case it would be something describing the bug that was fixed, i.e. "JSOCClient now works with non-time primekeys".

Reply:
:+1: